tests/logging_config/test_structlog_config.py

```diff
@@ -14,7 +14,7 @@ def test_structlog_config_default(capsys: CaptureFixture) -> None:
     with create_test_client([], logging_config=StructLoggingConfig()) as client:
         assert client.app.logger
         assert isinstance(client.app.logger, BindableLogger)
-        client.app.logger.info("message", key="value")
+        client.app.logger.info("message", key="value")  # type: ignore [attr-defined]
```

Comment:
What changed that this isn't typed correctly anymore?

Reply:
@provinzkraut this is the error that arises:
```
tests\logging_config\test_structlog_config.py:17: error: "BindableLogger" has no attribute "info"  [attr-defined]
```
This is not something I have changed.